oauth2-example-app/src/test/java/com/clouway/oauth2/exampleapp/storage/InMemoryClientRepositoryTest.java

```diff
@@ -8,10 +8,8 @@
  */
 public class InMemoryClientRepositoryTest extends ClientRepositoryContractTest {
 
-  private InMemoryClientRepository repository = new InMemoryClientRepository();
-
   @Override
-  public ClientRepository createRepository() {
-    return repository;
+  public ClientRepository createRepository(String defaultClientId) {
```

Comment:
the goal of that defaultClientId is that when i am going to test if client is registered i need  to know his id to retrieve it from repository, so i other words I pass the id of client thats going to be registered in constructor to make the test easier